libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/IpvSessionService.java

```diff
@@ -120,8 +120,13 @@ public IpvSessionItem generateIpvSession(
         if (errorObject == null) {
             if (isReverification) {
                 ipvSessionItem.pushState(new JourneyState(REVERIFICATION, START_STATE));
+                // Currently reverifcation journeys don't check the user's existing profile so we
+                // have to hard code this to P2 here. Eventually this can be set in the first step
+                // of the reverification journey.
+                ipvSessionItem.setTargetVot(Vot.P2);
             } else {
                 ipvSessionItem.pushState(new JourneyState(INITIAL_JOURNEY_SELECTION, START_STATE));
+                // For non-reverification journeys targetVot is set in CheckExistingIdentity
```

Comment:
I think this comment implies it's not done in the other case. Maybe an update to the comment above indicating it "updates" or "resets" targetVot?

Reply:
That implication is true - I guess these comments are confusing in some way then.

The other branch above is only for MFA reset (reverification) journeys and on those journeys the branch above is the only place that target vot is set.

What would you change in the comments to avoid the misunderstanding?

Reply:
Ah, we put people onto the reverification journey from InitialiseIpvSession. I was in the mindset we were setting the vot there (but I see your comment on that being difficult) so got confused